cmd/create.go

```diff
@@ -48,15 +55,18 @@ func checkCreateFlags(cmd *cobra.Command, args []string) error {
 		return errors.New("--skip-check-tag can only be used when --tag is specified")
 	}
 
-	if !createTag && !cmd.Flags().Changed("tag") {
-		return errors.New("Must specify a tag when create-tag=false")
+	skipTagCreation := useExistingTag || (!createTag)
+	if skipTagCreation && !cmd.Flags().Changed("tag") {
+		return errors.New("Must specify a tag when use-existing-tag=true")
```

Comment:
Oh, interesting. I was actually thinking of using it as `--use-existing-tag=<tag>` but semantically that would be incorrect. 

This solution is fine, defaults to false. 👍

Reply:
that's also a good idea that we can use -- let me think a little bit about it. I'm not sure where the existing flags came to be and don't want to break some flow I'm not aware of

Reply:
I actually prefer your suggestion. Let me think a bit about it